.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: php
 
-matrix:
-    include:
-        - php: hhvm
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-        - php: nightly
-    allow_failures:
-        - php: hhvm
-        - php: nightly
-    fast_finish: true
-
 sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
 
-install: composer install -n
+matrix:
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 5.6
+    - php: 5.6
+      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' 
```

Comment:
Not sure. Can you test that on your local machine and whether there is any difference to the 5.6 build?